xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.cpp

```diff
@@ -47,6 +42,7 @@ CActiveAESound::CActiveAESound(const std::string &filename) :
   m_isSeekPossible = false;
   m_fileSize = 0;
   m_isConverted = false;
+  m_activeAE = ae;
```

Comment:
use a reference to CActiveAE to make it explicit to the compiler and reader that it's valid for the lifetime of this class.